metaflow/metaflow_config.py

```diff
@@ -45,6 +45,13 @@ def from_conf(name, default=None):
 DEFAULT_DATASTORE = from_conf('METAFLOW_DEFAULT_DATASTORE', 'local')
 DEFAULT_METADATA = from_conf('METAFLOW_DEFAULT_METADATA', 'local')
 
+##
+# Per run configuration
```

Comment:
How is this per run configuration? These are now environment variables.  You could just say KFP configuration, simple, concise and conveys the intent of the code.

Reply:
Done.